scripts/tank_cmd.py

```diff
@@ -134,6 +148,18 @@ def show_help(log):
 ----------------------------------------------
 - To show this help, add a -h or --help flag.
 - To display verbose debug, add a --debug flag.
+- To provide a username and a password on the command line for authentication,
```

Comment:
We agreed about not allowing users to store login/password in the --credentials-file. From the beginning we've said that we would allow login/password on the command line.